hibernate-core/src/main/java/org/hibernate/internal/NaturalIdMultiLoadAccessStandard.java

```diff
@@ -110,7 +112,6 @@ public List<T> multiLoad(Object... ids) {
 			}
 
 			try {
-				session.autoFlushIfRequired( (Set) CollectionHelper.setOf( entityDescriptor.getQuerySpaces() ) );
```

Comment:
From the diff it looks as if this call went missing. I assume you removed it because it's already happening somewhere else. But I can't quite see where.

Reply:
This was on purpose

Reply:
OK good.